guillotina/files/utils.py

```diff
@@ -87,6 +86,4 @@ def guess_content_type(content_type, filename):
 
 
 def generate_key(context):
-    return "{}{}/{}::{}".format(
-        task_vars.container.get().id, get_content_path(context), context.__uuid__, uuid.uuid4().hex
-    )
+    return "{}/{}::{}".format(task_vars.container.get().id, context.__uuid__, uuid.uuid4().hex)
```

Comment:
I'm not sure if this change can break gcloud storage / s3 storage

Reply:
It shouldn't because all keys are stored persistently with the content(otherwise, moves for example wouldn't work); however, I wonder if we shouldn't change the default behavior here in case someone is depending on it?

Otherwise, this looks great!

Reply:
:+1: